package.json

```diff
@@ -96,7 +96,8 @@
     "vinyl-source-stream": "^2.0.0"
   },
   "overrides": {
-    "glob-parent": "6.0.2"
+    "glob-parent": "6.0.2",
+    "postcss": "^8.4.31"
   },
```

Comment:
There is not a standard method for adding comments to a `package.json` file, but some users have created a convention like this to add comments. Is this anything we are interested in exploring? 
```suggestion
  },
  "overridesComments": {
    "postcss": "This override should be removed once the related snyk error is resolved in @uswds/compile v1.1.0"
  },
```

Reply:
I like the idea of adding comments close to the code so we don't forget.  A new data field might not be necessary if we reference github issues in these types of commits. 

**My proposal**
1. Create issue for updating and removing override. 
2. Schedule issue two sprints out.
3. Add override and reference new issue in git commit message. Provides a nice reference in git blame.

**Git commit message**
```diff
- Add postcss override
+ Temporary postcss override. Remove with #ISSUE_NO
```

**Mockup**

You can see in the screenshot below. There's a line via git blame (light gray) and on hover we'll get a link to the issue.

<img width="400" alt="image" src="https://github.com/uswds/uswds-site/assets/3385219/2a536a11-ceef-46ff-8b5d-b4334b0eb18c">